packages/payment-detection/test/eth/fee-proxy-detector.test.ts

```diff
@@ -1,5 +1,9 @@
 import { AdvancedLogicTypes, PaymentTypes, RequestLogicTypes } from '@requestnetwork/types';
 import { EthFeeProxyPaymentDetector } from '../../src/eth/fee-proxy-detector';
+import TheGraphInfoRetriever from '../../src/erc20/thegraph-info-retriever';
+import { EVENTS_NAMES } from '@requestnetwork/types/dist/payment-types';
```

Comment:
```suggestion
import { PaymentTypes } from '@requestnetwork/types';
```

and use `PaymentTypes.EVENT_NAMES`

Reply:
Omg I'm importing from dist even 🤦 . Fixed in latest commit